README.rst

```diff
@@ -87,18 +103,19 @@ The easiest way to install Nansat on a Linux machine is to use
     cd <nansat_repository_folder>
     nosetests -w . --with-coverage --cover-package=nansat
 
-Fore more information see
-`Install-Nansat <https://github.com/nansencenter/nansat/wiki/Install-Nansat>`__
-section or use pre-configure virtual machines as explained on
-`Nansat-lectures <https://github.com/nansencenter/nansat-lectures>`__
 
 Usage
 -----
 
-.. code:: python
+::
 
     # download a test file
-    !wget https://github.com/nansencenter/nansat/raw/develop/nansat/tests/data/stere.tif
+    wget https://github.com/nansencenter/nansat/raw/develop/nansat/tests/data/stere.tif
+
+    # start python and work with Nansat
+    python
+
```

Comment:
Maybe it would be more logical to show here how to activate the environment?
```
# download a test file
wget https://github.com/nansencenter/nansat/raw/develop/nansat/tests/data/stere.tif
# activate the environment
source activate py3nansat
# launch python
python
```

```
# In Python interpreter:
# the rest of the code
```

Reply:
done